### ISSUE TITLE
Update documentation to more clearly describe how to programmatically set height or width in a ResizableBox

### ISSUE THREAD

**Issue by vullnetyy** [STRML/react-resizable#39]
I use react-resizable to resize a div vertically and the way I was trying to programmatically set the width and height of the box was to force it's parent to unmount and mount it again by changing the key like this: <ResizableBox ... key={this.props.height} />

This however gave me the following console error on every drag and drop resize: Uncaught Invariant Violation: findDOMNode was called on an unmounted component.

While experimenting, I later realized that the only thing you need to do is set the height and/or width as a css declaration (inline javascript style in my case) and everything was fine. This line solved everything: 'style={{height: this.state.height}}'

I can finally programmatically set the height of the component using a textbox.

My component now looks something like this

```
<ResizableBox 
    width={defaultWidth}
    height={this.props.height}
    onResizeStop={(e, obj) => this.onResizeStop(e, obj)}
    onResize={(e, obj) => this.onResize(e, obj)}
    draggableOpts={{onMouseDown:(e) => e.preventDefault()}/*to prevent dragging the whole component*/}
    style={{height: this.state.height}}
></ResizableBox>
```

**Comment by JoaoMosmann** [comment]
Actually the problem is that **ResizableBox** doesn't react for width and height changes. It only gets the initial value and store into its state.

I solved this problem by adding a **componentWillReceiveProps** to the **ResizableBox**. 

There is a PR waiting for approval. #44

**Comment by STRML** [comment]
This is fixed in `1.5.0`.
